packages/core/ios/RNSentry.h

```diff
@@ -14,7 +14,7 @@
 typedef int (*SymbolicateCallbackType)(const void *, Dl_info *);
 
 @interface SentryDebugImageProvider ()
-- (NSArray<SentryDebugMeta *> * _Nonnull)getDebugImagesForAddresses:(NSSet<NSString *> * _Nonnull)addresses isCrash:(BOOL)isCrash;
+- (NSArray<SentryDebugMeta *> * _Nonnull)getDebugImagesForImageAddressesFromCache:(NSSet<NSString *> * _Nonnull)addresses;
```

Comment:
When [the updated header file(`SentryDebugImageProvider+HybridSDKs.h`)](https://github.com/getsentry/sentry-cocoa/pull/4460) is released in the next cocoa version we may remove this declaration completely.